system/security/securesocket/securesocket.cpp

```diff
@@ -1824,9 +1823,7 @@ SECURESOCKET_API ISecureSocketContext* createSecureSocketContext(SecureSocketTyp
     }
     else
     {
-        if(server_securesocket_context.get() == NULL)
-            server_securesocket_context.setown(new securesocket::CSecureSocketContext(sockettype));
-        return server_securesocket_context.getLink();
+        return new securesocket::CSecureSocketContext(sockettype);
```

Comment:
Can we get rid of the if entirely?  Both outcomes now seem to look the same.

Reply:
yes, I should have noticed. Can also get rid of the CS which was only there to protect the global.
Changed.